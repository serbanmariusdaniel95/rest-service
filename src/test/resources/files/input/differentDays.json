{
  "displayed_what" : "Casa Ferlin",
  "displayed_where" : "Stampfenbachstrasse 38",
  "opening_hours" : {
    "days": {
      "monday": [
        {
          "start": "08:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "15:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ],
      "tuesday": [
        {
          "start": "09:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "16:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ],
      "wednesday": [
        {
          "start": "10:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "18:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ],
      "thursday": [
        {
          "start": "11:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "17:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ],
      "saturday": [
        {
          "start": "12:30",
          "end": "14:00",
          "type": "OPEN"
        },
        {
          "start": "18:30",
          "end": "22:00",
          "type": "OPEN"
        }
      ]
    }
  }
}