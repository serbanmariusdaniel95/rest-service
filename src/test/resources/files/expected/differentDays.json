{
  "name" : "Casa Ferlin",
  "address" : "Stampfenbachstrasse 38",
  "openingHours" : {
    "Monday": [
      "08:30 - 14:00",
      "15:30 - 22:00"
    ],
    "Tuesday": [
      "09:30 - 14:00",
      "16:30 - 22:00"
    ],
    "Wednesday": [
      "10:30 - 14:00",
      "18:30 - 22:00"
    ],
    "Thursday": [
      "11:30 - 14:00",
      "17:30 - 22:00"
    ],
    "Friday": [
      "closed"
    ],
    "Saturday": [
      "12:30 - 14:00",
      "18:30 - 22:00"
    ],
    "Sunday": [
      "closed"
    ]
  }
}